### PR TITLE
Proof Mechanisms advanced concepts section

### PR DESCRIPTION
**Pull request by CamrynCarter**
Section on Simple Hash and Khipu added to the advanced concepts.

### CODE REVIEW

File: content/docs/overview/advanced-concepts/index.md [hunk 1]
```diff
@@ -116,6 +116,20 @@ Once committed to the RKVST system, each lifecycle event record carries 2 separa
...
+
+When [creating an asset](../../rkvst-basics/creating-an-asset/), you may choose between two proof mechanisms, which will be used for that asset and its events. This will determine how your data is recorded on the RKVST shared blockchain.
+
+### Simple Hash
+
+The first option is Simple Hash. Simple Hash takes all the events within a past time period (the default is the last 30 days) and commits them to the blockchain as one hash. This hash value can then be used to compare the current state of the asset, and identify if any changes have occurred. With Simple Hash, you will not be able to see exactly what those changes were, only that something has changed. The Simple Hash proof mechanism is available with [all tiers](https://www.rkvst.com/pricing/) of the RKVST platform. 
+
+### Khipu
+
+The second option is Khipu. With Khipu, all the details of your asset and events are committed to the RKVST blockchain. This way, you may compare to previous versions of your asset and identify exactly what changes have been made, any time after the latest event is confirmed. The Khipu proof mechanism is available on our [Team and Enterprise tiers](https://www.rkvst.com/pricing/) of RKVST.
```
Comment (kjc-jitsuin) [inline content/docs/overview/advanced-concepts/index.md:131]: I think we should put the availability of this feature more obvious. Can we put the teams/enterprise piece in a note, or make it more visible?

Comment (CamrynCarter) [reply content/docs/overview/advanced-concepts/index.md]: yep, added as notes so they stand out

**Review by kjc-jitsuin — APPROVED** [review]

File: content/docs/overview/advanced-concepts/index.md [hunk 2]
```diff
@@ -116,6 +116,28 @@ Once committed to the RKVST system, each lifecycle event record carries 2 separa
...
+
+The first option is Simple Hash. Simple Hash takes all the events within a past time period (the default is the last 30 days) and commits them to the blockchain as one hash. This hash value can then be used to compare the current state of the asset, and identify if any changes have occurred. With Simple Hash, you will not be able to see exactly what those changes were, only that something has changed. 
+
+{{< note >}}
+**Note:** The Simple Hash proof mechanism is available with [all tiers](https://www.rkvst.com/pricing/) of the RKVST platform. 
+{{< /note >}}
+
+### Khipu
+
+The second option is Khipu. With Khipu, all the details of your asset and events are committed to the RKVST blockchain. This way, you may compare to previous versions of your asset and identify exactly what changes have been made, any time after the latest event is confirmed.
```
Comment (honourfish) [inline content/docs/overview/advanced-concepts/index.md:135]: `Any  time after the latest event is confirmed`

AFAIK it doesn't need to be gated by the latest event being confirmed. I could have an event a year ago that has been confirmed for a year, and check exactly what way it has been tampered with, if its been tampered with.

File: content/docs/overview/advanced-concepts/index.md [hunk 3]
```diff
@@ -116,6 +116,28 @@ Once committed to the RKVST system, each lifecycle event record carries 2 separa
 
 For more detailed information on Assets and how to implement them, please refer to [the Events API Reference](../../api-reference/events-api/).
 
+## Proof Mechanisms
+
+Assets and Events are core to the RKVST platform, and being able to quickly demonstrate proof that these artifacts have not been tampered is key to being able to use them.
+
+When [creating an asset](../../rkvst-basics/creating-an-asset/), you may choose between two proof mechanisms, which will be used for that asset and its events. This will determine how your data is recorded on the RKVST shared blockchain.
```
[the remark this answers is not in the document]
Comment (CamrynCarter) [reply content/docs/overview/advanced-concepts/index.md:123]: Changed to RKVST blockchain to match wording throughout sections

File: content/docs/overview/advanced-concepts/index.md (hunk 2 above)
Comment (CamrynCarter) [reply content/docs/overview/advanced-concepts/index.md:135]: Good to know, I removed that phrase to avoid confusion. Thanks!

**Pull request merged**